src/server.js

```diff
@@ -57,13 +58,12 @@ const startServer = async (portNumber) => {
     app.get("/setPassword/:token", (req, res) => {
       res.render("setPassword", { username: req.session.username });
     });
-    app.get("/databases", (req, res) => {
-      res.render("databases", { username: req.session.username });
-    });
     app.get("/resetPassword", (req, res) => {
       res.render("resetPassword", { username: req.session.username });
     });
     app.get("/postgres", postgres);
+    app.get("/mongodb", mongodb);
```

Comment:
rather than doing this, it should be refactored to 

```
const {database} = require('./routes/renderRoutes')
...
app.get("/database", database)
```